apps/api/src/domain/models/spatial-feature/entities/point.entity.ts

```diff
@@ -0,0 +1,57 @@
+import { RegisterIndexScopedCommands } from '../../../../app/controllers/command/command-info/decorators/register-index-scoped-commands.decorator';
+import { InternalError } from '../../../../lib/errors/InternalError';
+import cloneToPlainObject from '../../../../lib/utilities/cloneToPlainObject';
+import { DTO } from '../../../../types/DTO';
+import { AggregateCompositeIdentifier } from '../../../types/AggregateCompositeIdentifier';
+import { ResourceType } from '../../../types/ResourceType';
+import { Resource } from '../../resource.entity';
+import { IGeometricFeature } from '../interfaces/geometric-feature.interface';
+import { ISpatialFeature } from '../interfaces/spatial-feature.interface';
+import { PointCoordinates } from '../types/Coordinates/PointCoordinates';
+import { GeometricFeatureType } from '../types/GeometricFeatureType';
+import validatePosition2D from '../validation/validatePosition2D';
+
+@RegisterIndexScopedCommands([])
+export class Point extends Resource implements ISpatialFeature {
+    readonly type = ResourceType.spatialFeature;
+
+    readonly geometry: IGeometricFeature<typeof GeometricFeatureType.point, PointCoordinates>;
+
+    constructor(dto: DTO<Point>) {
+        super({ ...dto, type: ResourceType.spatialFeature });
+
+        if (!dto) return;
+
+        const { geometry: geometryDTO } = dto;
+
+        /**
+         * Do we want a class instead of a type for this property? Either way,
```

Comment:
Actually, I think we have answered this in the negative now. Update the comments.

Reply:
done